src/Symfony/Component/Serializer/Normalizer/ProblemNormalizer.php

```diff
@@ -40,20 +41,24 @@ public function __construct(bool $debug = false, array $defaultContext = [])
      *
      * @return array
      */
-    public function normalize($exception, $format = null, array $context = [])
+    public function normalize($object, $format = null, array $context = [])
     {
+        if (!$object instanceof FlattenException) {
```

Comment:
Maybe could you just add a `@param` tag here to make Psalm happy?

Reply:
I can remove these lines if preferred ; this does not have impact psalm for developer which extends this class.
My main concern was about the name of the param.

This method is public, and the interface is allowing `object` ; so theorically this method should handle every possible object and not restricting the param directly in the phpdoc.

I agree that this seems duplicate with the `supportNormalization` method. We could consider that this is similar to Voter https://github.com/symfony/symfony/blob/72b94b8fb7fe0f75e6867ac2f8552c7ce58d6ccf/src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php#L74.

But when I look at others Normalizer, it seems that checking the param was the strategy adopted
- https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php#L53
- https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/Serializer/Normalizer/ArrayDenormalizer.php#L42
- https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/Serializer/Normalizer/JsonSerializableNormalizer.php#L33
- https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/Serializer/Normalizer/DataUriNormalizer.php#L55

So I added consistency